app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -554,14 +533,60 @@ class AdvancedFilter extends React.Component {
             {filterOption?.type === 'number' && (
               <Form.Group className="py-0 my-0">
                 <Row>
-                  <Col md="auto">{this.renderOperatorOptions(operatorOption, index, value)}</Col>
+                  {filterOption?.options && (
+                    // specific dropdown for filters with number type but require additional options
+                    <Col md="8">
+                      <Form.Control
+                        as="select"
+                        value={value.option}
+                        onChange={event =>
+                          this.changeValue(index, {
+                            number: value.number,
```

Comment:
Thoughts on adding a optional chaining operator to these just in check?

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/517/commits/3c987ae6eac329381d50f54916b876169156fb2a#diff-d7415a139df13ce270099f966bce0848765150673d2ce9865a7e0c4d9d87834fR544